core/src/main/resources/application-dev.yml

```diff
@@ -1 +1,2 @@
-cb.image.catalog.url: https://cloudbreak-imagecatalog.s3.amazonaws.com/v3-dev-cb-image-catalog.json
\ No newline at end of file
+cb.image.catalog.url: https://cloudbreak-imagecatalog.s3.amazonaws.com/v3-dev-cb-image-catalog.json
+cb.aws.externaldatabase.retentionperiod: 0
```

Comment:
I think it can be part of the simple application yml and then we will rewrite in help charts

Reply:
fine for me.